spec/spec_helper.rb

```diff
@@ -13,6 +13,8 @@
 
 Sidekiq::Logging.logger = Logger.new('log/sidekiq-test.log')
 
+load "#{Rails.root}/db/seeds.rb"
```

Comment:
If the tests break otherwise, leave it. We can remove it when we found out why we need it.